### ISSUE TITLE
Disable `:unused-bindings` warning for as-bindings like `{:as my-symbol :keys [...]}`?

### ISSUE THREAD

**Issue by theronic** [clj-kondo/clj-kondo#2348]
clj-kondo raises an "unused binding" warning for as-bindings when destructuring,e.g.
```
{:as test-spice :keys [client dispose]} (permissions.spicedb.testing/make-test-client)
```
raises:
```
in test/api/server_test.clj:488:16: warning: unused binding test-spice
```
Is there a way to disable this for as-bindings? I intentionally leave as-bindings in so the reader knows what generic destructured keys mean.

**Comment by borkdude** [comment]
See https://github.com/clj-kondo/clj-kondo/blob/master/doc/linters.md#unused-binding

```
{:linters {:unused-binding {:exclude-destructured-as true}}}
```

**Comment by borkdude** [comment]
What I also sometimes to is start the `:as` name with an underscore: `:as _foo`
